packages/component-library/styles/layers.scss

```diff
@@ -1,8 +1,8 @@
 $ca-z-index-body: 0;
-$ca-z-index-dropdown: 1000;
 $ca-z-index-sticky: 1020;
 $ca-z-index-fixed: 1030;
 $ca-z-index-modal-backdrop: 1040;
 $ca-z-index-modal: 1050;
+$ca-z-index-dropdown: 1055;
```

Comment:
LMK if this is alright. I'm not sure what all the situations are, but I feel like a `dropdown` is more closely related to a `popover`, in terms of layer positioning.

The main thing we need in perform is to have the z-index > 1030 (above the `fixed` layer`).

Reply:
@ActuallyACat and @sentience may be interested in this too.

Reply:
I got the ok from @ActuallyACat while we were on slack. Confirming if this update is alright on your end @sentience

Reply:
Makes sense to me. As we've discussed before (I think), a global z-index scale is not a panacea, as stacking contexts often mean that `z-index` becomes locally scoped, at which point the scale falls apart.

With that caveat, I think that putting dropdown explicitly on top of fixed elements and modals makes perfect sense, where a global scale can be used.